backend/src/main/kotlin/be/osoc/team1/backend/entities/Student.kt

```diff
@@ -130,26 +135,58 @@ class Student(
 
     @ManyToMany(cascade = [CascadeType.MERGE])
     @OrderBy
+    @field:JsonView(StudentView.Full::class)
     val skills: Set<Skill> = sortedSetOf(),
+
+    @field:JsonView(StudentView.Basic::class)
     val alumn: Boolean = false,
+
+    @field:JsonView(StudentView.Full::class)
     val possibleStudentCoach: Boolean = false,
+
+) {
+
     @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true)
+    @field:JsonView(StudentView.Full::class)
     @JsonSerialize(using = AnswerListSerializer::class)
-    val answers: List<Answer> = listOf()
-) {
+    var answers: List<Answer> = listOf()
```

Comment:
Why did you move this out of the constructor?

Reply:
For some reason, arguments in the constructor that have a jsonview and jsonserializer won't be serialized in the response. I have found nothing about this online, so I moved it out of the constructor. I know it isn't optimal, but I couldn't find another solution

Reply:
I guess it's not too bad to keep it like this.